core/templates/dev/head/pages/base.html

```diff
@@ -72,6 +72,8 @@
       {% include 'pages/header_css_libs.html' %}
     {% endblock header_css %}
 
+    <script src="{{TEMPLATE_DIR_PREFIX}}/pages/load_constants.js"></script>
```

Comment:
Possibly related: http://stackoverflow.com/questions/16286605/angularjs-initialize-service-with-asynchronous-data. Could maybe bootstrap Angular only after the constants have loaded -- but @vojtechjelinek, do you know whether this is an issue in the first place?